### DISCUSSION TITLE
Bun is good idea?

### DISCUSSION THREAD

**Discussion by tavaresgerson** [adonisjs/core#3938] [General]
Hello people,

I see Bun ([bun.sh](https://bun.sh)) project and your benchmark is incredible. So, do you usage in your projects for production? Is this recommended? :thinking: 

Thanks

**Comment by McSneaky** [comment]
I'd say it's too early 🙂 

I've been using Deno (stable) with some projects and I'd say even this is too young. There's a lot of missing docs, cornercases etc
Bun is too early to use in prod, it's still in beta, let it become stable first

Also there's interesting runtime: `just-js` but that's too unstable too to use in prod
Just benchmark compared to C: https://www.techempower.com/benchmarks/#section=data-r21&l=zik0se-6bj
And Just repo: https://github.com/just-js/just

There are interesting runtimes growing to play around, test, **contribute** and see where they grow in a year or two

You really don't want your prod to go down coz of some segfault in runtime and then have to fix low level runtime code to get your product back up and running

**Reply by tavaresgerson** [reply]
Wow! 

Didn't know `just-js`, thanks for your comment

`just-js` is awesome, you're right, better wait a little longer.
